src/components/contentGetters/services/category.service.js

```diff
@@ -65,6 +102,18 @@ angular.module('znk.infra.contentGetters').service('CategoryService',
             });
         };
 
+        self.getCategoryLevel1ParentSync = function (category) {
```

Comment:
Check if anyone is using this func, I believe it's not needed, use getCategoryLevel1ParentByIdSync instead